formulae/include/python3.6m

```diff
@@ -0,0 +1 @@
+/usr/include/python3.6m
```

Comment:
Should this file be included? If so, what is its purpose? If not, then remove it from your branch and add it to the `.gitignore` file.